invoice/invoice.go

```diff
@@ -0,0 +1,33 @@
+package invoice
+
+import (
+	"context"
+	"errors"
+	"proxy/service"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lntypes"
+)
+
+func GenerateInvoice(svc *service.Service, ctx context.Context, lnInvoice lnrpc.Invoice) (string, lntypes.Hash, error) {
```

Comment:
Mentioned elsewhere, should be defined on `svc` I think, it's not useful to have a package which has a single function.